CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ For details about compatibility between different releases, see the **Commitment
 
 ### Changed
 
+- Disconnect gateways that are deleted on the Identity Server.
```

Comment:
I think `Gateways which have been deleted are now automatically disconnected by the Gateway Server` sounds a bit better.